localstack/aws/protocol/service_router.py

```diff
@@ -96,6 +100,10 @@ def _extract_service_indicators(request: Request) -> _ServiceIndicators:
         "/2015-01-01": "es",
         "/2021-01-01": "opensearch",
     },
+    "sagemaker": {
+        "/endpoints/": "sagemaker-runtime",
```

Comment:
Does it make a difference here if we add `"/endpoints/"` instead of `"/endpoints"` as discussed in the internal discussions?

Reply:
it's a little stricter which can help to avoid future conflicts. since there is no operation `/endpoints` but only `/endpoints/<path>` it made sense to me. i guess the drawback is that if they add a list endpoints operation to sagemaker-runtime then this will not route it correctly :-)

Reply:
i changed my mind and changed it to `/endpints`

Reply:
I don't think that there ever will be something other than invocations/executions in that API (famous last words), so it's not really that important. I'm fine with both.